setup/autoinstall-iso/iso-change.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -x
```

Comment:
```suggestion
set -x -e
```
Kills the script upon error at one line. I find this nice so that commands that fail cause the entire script to fail so the user can fix the issue instead of proceeding in a broken state